exporter/elasticsearchexporter/model.go

```diff
@@ -58,9 +73,47 @@ func (m *encodeModel) encodeLog(resource pcommon.Resource, record plog.LogRecord
 		document.Sort()
 	}
 
-	var buf bytes.Buffer
-	err := document.Serialize(&buf, m.dedot)
-	return buf.Bytes(), err
+	return document
+
+}
+
+func (m *encodeModel) encodeLogECSMode(resource pcommon.Resource, record plog.LogRecord, scope pcommon.InstrumentationScope, now time.Time) objmodel.Document {
+	var document objmodel.Document
+
+	// First, try to map resource-level attributes to ECS fields.
+	resourceAttrsConversionMap := map[string]string{
```

Comment:
I think what's missing is the translation of `k8s.*` SemConv resource attributes. See also https://github.com/elastic/apm-data/blob/0e072ceaadae2ef8f61d75c3a27c47cc5ce81257/input/otlp/metadata.go#L145-L165

Reply:
Yep, I didn't include them as they're not really part of ECS.  But as per the off-PR discussion with @axw and you, I will include them now for practical reasons.

Reply:
Added in 468043f049.

Reply:
Done in 468043f0494859176c0fd89d3c1d6c62466068d2.